src/components/task-plan/reading.cjsx

```diff
@@ -38,7 +40,22 @@ SelectTopics = React.createClass
     return
 
   render: ->
-    <BS.Button onClick={@handleToggle} bsStyle="primary">Edit Readings</BS.Button>
+    if (@props.selected.length and TocStore.isLoaded())
```

Comment:
Hm, if it errors, the button will likely not display...we may need an on error handler.  We likely need to think about error handling as a whole for that app.

I'll add an error container though.